WooCommerce/Classes/ViewRelated/Orders/Order Details/Address Edit/EditAddressFormViewModel.swift

```diff
@@ -4,6 +4,19 @@ import Combine
 
 final class EditAddressFormViewModel: ObservableObject {
 
+    enum AddressType {
+        case shipping
+        case billing
+    }
```

Comment:
I was thinking if instead of having an enum, it would be a good idea to have a protocol. 

That way: we could reuse this screen to any other possible address form in the app. Not sure if it's worth it tho. Do you think we should add a "card" to tackle it leave an issue later?

Reply:
Yes, moving all this stuff to protocol should work. I’ll make a quick draft on Monday to check this idea.